lib/transform.js

```diff
@@ -156,6 +156,9 @@ function transform(source, template, options = {}) {
   let classNameBindings = findClassNameBindings(properties);
   debug('classNameBindings: %o', classNameBindings);
 
+  let ariaRole = findAriaRole(properties);
```

Comment:
This only returns the `path`, so you would need to create and add the (template) AST nodes to `attrs` here: https://github.com/ember-codemods/tagless-ember-components-codemod/pull/51/files#diff-9000d2ea35a5fd71d0a8caffed685d47R207